components/canvas/Cargo.toml

```diff
@@ -11,6 +11,10 @@ rust-version.workspace = true
 name = "canvas"
 path = "lib.rs"
 
+[features]
+vello = ["dep:vello", "dep:pollster", "dep:futures-intrusive"]
+default = ["vello"]
```

Comment:
This needs to go before landing:

```suggestion
```

we also need to make sure that servoshell can enable vello feature.